docs/install/arm.md

```diff
@@ -51,7 +49,8 @@ curl https://raw.githubusercontent.com/prysmaticlabs/prysm/master/prysm.sh --out
 ./prysm.sh beacon-chain
 ```
 
-It is also recommended to include the `--p2p-host-ip` and `--min-sync-peers 7` flags to improve peering. For advanced users that desire standard debugging tools found in the Busybox base image, append a `--debug` flag to enable them.
+> Advanced users: Please refer to the article on how to [improve P2P connectivity](../p2p-host-ip).  
```

Comment:
a general note: its much better to use the full root path when linking within documentation (ie /docs/install/blahblah), that way even if they are reorganised there is no breakage

Reply:
fixed in a few spots, will note for later PRs